src/main/java/org/openstreetmap/atlas/vectortiles/TippecanoeSettings.java

```diff
@@ -0,0 +1,50 @@
+package org.openstreetmap.atlas.vectortiles;
+
+import java.util.function.BiConsumer;
+
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasEntity;
+
+import com.google.gson.JsonObject;
+
+/**
+ * This is where you configure the tippecanoe properties that will be in the generated
+ * line-delimited GeoJSON. You also set the arguments to drive tippecanoe here.
+ *
+ * @author hallahan
+ */
+final class TippecanoeSettings
+{
+    private TippecanoeSettings()
+    {
+        // Utility Class
+    }
+
+    static final DefaultArtifactVersion MIN_VERSION = new DefaultArtifactVersion("1.32.1");
+    static final String GEOJSON = "EVERYTHING.geojson";
+
+    static final String[] ARGS = new String[] { "-Z6", "-z14", "--generate-ids", "--read-parallel",
+            "--no-tile-size-limit", "--no-feature-limit" };
+
+    static final BiConsumer<AtlasEntity, JsonObject> JSON_MUTATOR = (atlasEntity, feature) ->
```

Comment:
This should be done using a Json configuration file in Resources, and a `ConfiguredTaggableFilter`. [Documentation here](https://github.com/osmlab/atlas/tree/4f20c144cdf5f46f59a4947410d1aa50d4be5211/src/main/java/org/openstreetmap/atlas/tags/filters#tag-filtering)!

Reply:
We decided to leave this and make it public. Making this a JSON config totally makes sense though, could be a good optimization.